types/node/globals.d.ts

```diff
@@ -702,9 +702,9 @@ declare namespace NodeJS {
         readonly writableLength: number;
         columns?: number;
         rows?: number;
-        _write(chunk: any, encoding: string, callback: Function): void;
-        _destroy(err: Error | null, callback: Function): void;
-        _final(callback: Function): void;
+        _write(chunk: any, encoding: string, callback: (err?: Error) => void): void;
```

Comment:
```suggestion
        _write(chunk: any, encoding: string, callback: (err?: Error | null) => void): void;
```
see `class Writable`